.github/workflows/main.yml

```diff
@@ -136,12 +141,29 @@ jobs:
         run: sudo dpkg -i /var/cache/apt/archives/*.deb
 
       - name: Install Lua packages
+        if: matrix.lua_version != "5.4"
         run: |
           if [ "${{ matrix.lua_name }}" = "luajit" ]; then
             sudo apt-get install libluajit-5.1-dev luajit
           fi
           sudo apt-get install liblua${{ matrix.lua_version }}-dev lua${{ matrix.lua_version }}
 
+      # Ubuntu 20.04 hasn't a package for Lua 5.4, we need to build it from source.
+      # (Note that we need first to install curl and tar to download the latest Lua release)
+      - name: Build and Install Lua 5.4
+        if: matrix.lua_version == "5.4"
+        run: |
+          sudo apt-get install --download-only -y --no-install-recommends \
+            curl \
+            tar
+          sudo dpkg -i /var/cache/apt/archives/*.deb
+          curl -R -O http://www.lua.org/ftp/lua-5.4.3.tar.gz
+          tar zxf lua-5.4.3.tar.gz
+          cd lua-5.4.3
```

Comment:
It should probably be fine, but this does "pollute" the CWD.
The other steps all do that in `/tmp`, which also makes setting the cache path easier.